package.json

```
@@ -32,6 +32,8 @@
     "cp:spec": "./scripts/content-modules/cp-pages.sh",
     "diff:check": "git diff --name-only --exit-code || (echo; echo 'WARNING: the files above have not been committed'; echo)",
     "diff:fail": "git diff --name-only --exit-code || (echo; echo 'ERROR: the files above have changed. Locally rerun `npm run test` and commit changes'; echo; git diff | head -100; exit 1)",
+    "format": "npm run _check:formatting -- --write",
+    "format:write": "npm run _check:formatting -- --write",
```

I'd vote for at most one script alias (I'd even eliminate `prettier:write`, but I can do that in a followup PR since there's some cleanup I'd like to do at the same time):

```suggestion
    "format": "npm run _check:formatting -- --write",
```